### DISCUSSION TITLE
blog/2020/07/26/capture_screen_using_gdiplus

### DISCUSSION THREAD

**Discussion by giscus** [SuperKogito/SuperKogito.github.io#7] [Announcements]
# blog/2020/07/26/capture_screen_using_gdiplus

Capturing the screen using Gdiplus

https://superkogito.github.io/blog/2020/07/26/capture_screen_using_gdiplus.html

**Comment by Blekie** [comment]
The provided code reeks from leaks... If there is anything more to be done please add it to the code in git or here... 
After 
 DeleteDC(hwindowCompatibleDC);
 ReleaseDC(hWnd, hwindowDC);
without save functionality.  
Thankfully closing the program cleans the memory. But if i loop capturing 1920x1080 my pc dies in seconds.

**Reply by SuperKogito** [reply]
The code is provided as it is, without any claims on its efficiency. At the time of writing the code worked without issues on my machine. The code is available on [gitgist](https://gist.github.com/SuperKogito/00e0ad0d5b2b567d74a10fe18c048776). You are welcome to improve/edit the code and post a newer version as a comment on gist for future users.

As for your issue, the code has no looping of the screeshooting, so when you loop that make sure to clean the memory after each iteration to avoid leaks. Good luck.
